cli/js/web/dom_types.ts

```diff
@@ -277,6 +277,9 @@ export interface Blob {
   readonly size: number;
   readonly type: string;
   slice(start?: number, end?: number, contentType?: string): Blob;
+  stream(): ReadableStream;
+  text(): Promise<string>;
+  arrayBuffer(): Promise<ArrayBuffer>;
```

Comment:
Do these need to be added somewhere else?

These methods work for me, but seem to require `// @ts-ignores` to compile.